framework/core/js/src/admin/components/UserListPage.tsx

```diff
@@ -91,6 +95,18 @@ export default class UserListPage extends AdminPage {
     const columns = this.columns().toArray();
 
     return [
+      <div className="Search-input">
+        <input
+          className="FormControl SearchBar"
+          type="search"
+          placeholder={app.translator.trans('core.admin.users.search_placeholder')}
+          oninput={(e: InputEvent) => {
+            this.isLoadingPage = true;
+            this.query = (e?.target as HTMLInputElement)?.value;
+            debounce(250, () => this.loadPage(this.pageNumber))();
```

Comment:
the debounce isn't actually working right now.
That's because we create a new denounced function on every re-render, and the only way debounce works is if we use the same debounce function we create.

To amend that, we can set the denounced function on a private property then we can use that instead.

```ts
private throttledSearch = debounce(250, () => this.loadPage(0));
```